video/software/src/decoder/h263.rs

```diff
@@ -19,6 +20,12 @@ pub enum H263Error {
     MissingWidthHeight,
 }
 
+impl From<H263Error> for Error {
+    fn from(error: H263Error) -> Self {
+        Error::DecoderError(Box::new(error))
+    }
+}
+
```

Comment:
Let's name these simply `Error`, preferring to do e.to match Rust naming conventions of not repeating yourself, preferring `h263::Error`. (I know we've been bad at this so far!)

Reply:
The problem is that we have derive Error for Error and then impl Error on that Error. I had to name it something else to avoid specific path everywhere in the file.

Reply:
Sort of along the lines of the other comment, maybe this should be exported as `ruffle_video::Error` in the parent crate, in which case explicitly specifying the path is actually a little nice. There's also the possibility of using a renaming import here.

Don't think it's mega important, but I figure if we are re-organizing everything, we could take some time to think about the module layout and naming conventions.